docs/coding-standards.md

```diff
@@ -47,10 +47,34 @@ To validate SASS files use `lando gulp validate:sass`.
 * Maximum allowed compound selectors is increased from 3 to 4 (selector-max-compound-selectors)
 * Alphabetical order of properties is not enforced (order/properties-alphabetical-order)
 
+**Rule:** `max-nesting-depth`
+* Recommend setting is `1` but then, what is the point of SASS
+
+**Rule:** `selector-no-qualifying-type`
+* Qualifying a selector by type is allowed.
+
+**Rule** `selector-max-compound-selectors`
+* Maximum allowed compound selectors is increased from 3 to 4.
+
+**Rule** `order/properties-alphabetical-order`
+* Alphabetical order of properties is not enforced.
+
+To temporarily disable the linter for certain code blocks you can use the syntax
+below. More information can be found in the linter [docs][].
+
+```
+/* stylelint-disable selector-no-id, declaration-no-important  */
+#id {
+  color: pink !important;
+}
+/* stylelint-enable */
+```
```

Comment:
Also give an example of single line disable: 

```
.custom-select {
  -webkit-appearance: none; // stylelint-disable-line property-no-vendor-prefix
}```

Reply:
Added.